### PR TITLE
PCRM-0000 Adding TFSec to Code Scanning

### PR DESCRIPTION
**Pull request by joeds13**
Code Scanning of Terraform Stacks

### CODE REVIEW

File: .tfsec/config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+---
```
Comment (HarveyNHS) [inline .tfsec/config.yml:1]: What's this extra line for?

**Review by HarveyNHS — COMMENTED** [review]
Please review

File: .github/workflows/code-scanning.yml [hunk 2]
```diff
@@ -10,29 +10,61 @@ on:
...
+        uses: actions/checkout@v2
+
+      - name: Python Package
+        run: |
+          make python-package
+
+      - name: Terraform Init
+        run: |
+          make init
+          make workspace env="default"
```
Comment (jamesnunnnhs) [inline .github/workflows/code-scanning.yml:53]: Should this be default? or the branch id?

File: .tfsec/config.yml (hunk 1 above)
Comment (joeds13) [reply .tfsec/config.yml:1]: It's the YAML file header, we'll need this config file in a follow up PR to include our own checks

File: .github/workflows/code-scanning.yml (hunk 2 above)
Comment (joeds13) [reply .github/workflows/code-scanning.yml:53]: Using "default" as we just need an initialised terraform stack, we'll never run any further actions

**Review by abbas-khan8 — APPROVED** [review]

**Review by HarveyNHS — APPROVED** [review]
Approved

**Pull request merged**